recipes-extended/spi-minnowmax-board/spi-minnowmax-board.bb

```diff
@@ -4,6 +4,9 @@ LICENSE = "GPLv2"
 LIC_FILES_CHKSUM = "file://${THISDIR}/files/COPYING;md5=12f884d2ae1ff87c09e5b7ccc2c4ca7e"
 PR = "r0"
 
+# The module is compatible with intel-corei7-64/Minnowboard only
+COMPATIBLE_MACHINE="intel-corei7-64"
```

Comment:
I understand that COMPATIBLE_MACHINE just reduces the targets to build recipes for, so this should not make the behavior any worse. Nevertheless, does the recipe cause problems for other intel-corei7-64 targets than minnowmax?